VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -69,7 +69,7 @@
                 "genwzquarks"     : NTupleCollection("GenWZQuark",   genParticleType, 6, help="Generated quarks from W/Z decays"),
                 "genleps"         : NTupleCollection("GenLep",     genParticleType, 2, help="Generated leptons from W/Z decays"),
                 "gentauleps"      : NTupleCollection("GenLepFromTau", genParticleType, 6, help="Generated leptons from decays of taus from W/Z decays"),
-		"genHiggsBoson"   : NTupleCollection("GenHiggsBoson", genParticleType, 1, help="Generated Higgs boson "),
+		"genHiggsBoson"   : NTupleCollection("GenHiggsBoson", genParticleType, 2, help="Generated Higgs boson "),
```

Comment:
qui se metti 2 non cambia nulla a meno di prendere genHiggsBosons con la s in fondo no?